src/main/kotlin/anttracker/issues/Issues.kt

```diff
@@ -53,6 +131,17 @@ private fun selectIssueToViewMenu(rows: RowToIssuePage) =
         }
     }
 
+private fun viewIssueMenu(rows: RowToIssuePage) =
+    screenWithMenu {
+        title("View issue")
+        content { t ->
+            t.prompt(
```

Comment:
Probably once we got most of the modules working together we should do a sweep to see what UI differs from what's in the user manual.

Reply:
As we go, we should check. Otherwise, it will be a lot of potential work.